python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -87,8 +92,37 @@ def from_pretrained(cls,
         load_in_4bit = kwargs.pop("load_in_4bit", False)
         load_in_low_bit = kwargs.pop("load_in_low_bit", None)
         optimize_model = kwargs.pop("optimize_model", True)
+        user_quantization_config = kwargs.pop("quantization_config", None)
 
         if load_in_4bit or load_in_low_bit:
+
+            if config_dict.get("quantization_config", None) is not None:
+                from bigdl.llm.transformers.low_bit_linear import get_ggml_qk_size
+                q_config = config_dict["quantization_config"]
+                if q_config["quant_method"] == "gptq":
+                    invalidInputError(q_config["bits"] == 4,
+                                      "Only 4-bit gptq is supported in bigdl-llm.")
+                    invalidInputError(q_config["desc_act"] is False,
+                                      "Only desc_act=False is supported in bigdl-llm.")
+                    if load_in_low_bit is not None:
+                        invalidInputError(load_in_low_bit == "asym_int4",
+                                          "You can only load gptq model as aysm_int4 low bit type.")
+
+                    load_in_low_bit = "asym_int4"
+                    if int(q_config["group_size"]) % get_ggml_qk_size(load_in_low_bit) != 0:
+                        invalidInputError(False,
+                                          (f"group_size must be divisible by "
+                                           f"{get_ggml_qk_size(load_in_low_bit)}."))
+                    if user_quantization_config is not None:
+                        invalidInputError(user_quantization_config.bits == 4,
+                                          "Only 4-bit gptq is supported in bigdl-llm.")
+                        invalidInputError(user_quantization_config.use_exllama is False,
+                                          "Only use_exllama=False is supported in bigdl-llm.")
+                    else:
+                        from transformers import GPTQConfig
+                        user_quantization_config = GPTQConfig(bits=4, use_exllama=False)
+                    kwargs["quantization_config"] = user_quantization_config
```

Comment:
Does save/load low bit work? Do we need to remove `quantization_config` in `save_low_bit`?

Reply:
save/load low bit works. It seems our `load_low_bit` will ignore `quantization_config`, but I remove it anyway.